### DISCUSSION TITLE
how you guys contact github's staff? No response for my ticket for 10 days.

### DISCUSSION THREAD

**Discussion by ziyigogogo** [community/community#36530] [Copilot Conversations]
I opened a ticket like 10 days ago here, https://support.github.com/ticket/personal/0/1824906

In my country, copilot seems not work fluently because of the bad internet condition. So after a quickly trial about 3 or 4 days two months ago, I stoped using it. However I was charged today for a year($83 usd),therefor I’m asking for a refund for that. 

Is there any advice for me for this situation???

**Comment by iamcjwilliams** [comment]
@ziyigogogo We're sorry for the trouble here. Our copilot team is currently experiencing a high volume of tickets, so responses will be delayed. Our copilot team has been notified, I will follow up as soon as I have an update. Thank you for your patience
